### ISSUE TITLE
Black squares around objects when bloom rendering is enabled

### ISSUE THREAD

**Issue by MAG-AdrianMeredith** [playcanvas/engine#7105]
### Description

Certain platforms are shows black when bloom is enabled and looking at certain models.  This has been reproduced on 3 separate machines but is only on windows.  Tested against playcanvas 2.0, 2.1 and 2.2.2

[about-gpu-2024-10-15T15-11-55-300Z (1).txt](https://github.com/user-attachments/files/17730208/about-gpu-2024-10-15T15-11-55-300Z.1.txt)

https://github.com/user-attachments/assets/aa090eb9-8a31-4c76-bdd9-ecf7ebb2f1e3

### Steps to Reproduce

1. On windows with dedicated nvidia graphics
2. sign up to oko at app.oko.live
3. go to this space https://app.oko.live/space/6733807b8140be9b4af2ed90
4. toggle between bloom and not bloom in graphics settings shows a black screen
5. not reproducible on my mac (macbook pro m3)

**Comment by MAG-AdrianMeredith** [comment]
Once thing of note is the models themselves have a degenerate triangle https://sketchfab.com/3d-models/halo-mk-v-model-ff8a4e4f9d1f4fdeb3a1d887f6f70dae which could be related

**Comment by mvaligursky** [comment]
yeah it does seem that rendering in sketchfab adds those strange lines too - and I suspect they generate some incorrect values somewhere during rendering.

**Comment by mvaligursky** [comment]
Likely fixed by this: https://github.com/playcanvas/engine/pull/7717
I have not tested this specifically, but the solution there should handle most if not all cases.
Please reopen if you find it's not fixed by it, thanks.

**Comment by MAG-AdrianMeredith** [comment]
Oooh, I'l retest this soon.  thanks
